eitprocessing/datahandling/loading/draeger.py

```diff
@@ -88,9 +88,14 @@ def load_from_single_path(
                 previous_marker,
             )
 
+    estimated_sample_frequency = round((len(time) - 1) / (time[-1] - time[0]), 6)
+
     if not sample_frequency:
-        msg = "No sample frequency provided. "
-        raise ValueError(msg)
+        sample_frequency = estimated_sample_frequency
+
+    if sample_frequency != estimated_sample_frequency:
+        msg = "Provided sample frequency {} does not match the estimated sample frequency."
```

Comment:
It seems that the curly brackets aren't actually used in this warning message? It might also make sense to display both the provided sample frequency and the estimated sample frequency when this warning is triggered?

Reply:
Yes! Thanks for noticing. I fixed it in the latest commit.

I also changed the precision of the rounding; I saw for other values that 4-5 digits is the limit for 64 bit floats.